### PR TITLE
feat: Add combiner "ModulesInfo"

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### CODE REVIEW

**Comment by huali027** [comment]
@xiangce  Please help to review, thanks.

File: insights/combiners/modinfo.py [hunk 1]
```diff
@@ -1,14 +1,20 @@
 """
-ModInfo
```
Comment (ryan-blakley) [inline insights/combiners/modinfo.py:2]: I'm curious why you're removing the Modinfo documentation? The code is still there even though it's deprecated.

**Comment by huali027** [comment]
@xiangce @ryan-blakley  I recovered the doc for the deprecated classes, please help to review again, thanks.

**Review by ryan-blakley — APPROVED** [review]
Looks good to me.

**Review by xiangce — APPROVED** [review]

**Comment by xiangce** [comment]
@huali027 - I went ahead and merged this PR.  However, beside of the "git commit message" and the PR title.  please add a short description for the future PRs. Thanks.

**Pull request merged**